tests/device_tests/ethereum/test_signtx.py

```
@@ -536,8 +542,16 @@ def test_signtx_staking(
     )
 
 
-@parametrize_using_common_fixtures("ethereum/sign_tx_staking_data_error.json")
-def test_signtx_staking_bad_inputs(session: Session, parameters: dict, result: dict):
+@parametrize_using_common_fixtures(
+    "ethereum/sign_tx_staking_data_error.json",
+    "ethereum/sign_tx_error.json",
+)
+def test_signtx_error(session: Session, parameters: dict, result: dict):
```

Ruff (0.14.14)

[warning] Unused function argument: `result`

(ARG001)